app/assets/stylesheets/ama_layout/layout/sidebar.scss

```diff
@@ -1,104 +1,108 @@
-
 .dashboard-title {
   font-size: $base-font-size*0.875;
   font-weight: normal;
-  color: #5C5C5C;
+  color: $charcoal;
   text-transform: uppercase;
-  background:white;
-  padding: $base-margin*3;
+  background: $white;
+  padding: $base-padding*3;
 }
 
-.dashboard-nav {
+.dashboard-nav{
```

Comment:
If we're following Github's style guide: http://primercss.io/guidelines/#spacing, we should keep the space before `{`.